overlord/configstate/configcore/netplantest/netplantest.go

```diff
@@ -0,0 +1,139 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
I'm a bit unsure about the placement for this, it's going to be used only here but it's still a bit odd, maybe it should live under dbusutil?

Reply:
Happy to move it under dbusutil, either way is fine of course.

Reply:
Moved now, thanks for the suggestion.